### PR TITLE
Refactored out the NavButtons into separate components.

### PR DESCRIPTION
**Pull request by NicholasLYang**
Changed the table formatting and old school CSS for Flexbox and responsive units (vw and vh). In the future please try to use responsive units when possible.

Addresses #163

### CODE REVIEW

**Comment by NicholasLYang** [comment]
Instead of messing with the other PR, how about you look at this one @jsonkao?

File: src/js/modules/core/components/ClassifiedsRedirect.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default ClassifiedsRedirect = () => {
+  window.location = "https://goo.gl/forms/bltSaUMOwVW5TpN02";
+  return (
+    <p>
+      If you're not automatically redirected, please{" "}
```
Comment (jsonkao) [inline src/js/modules/core/components/ClassifiedsRedirect.js]:
```suggestion
      If you&rsquo;re not automatically redirected, please{" "}
```

File: src/js/modules/core/components/SectionsButton.js [hunk 2]
```diff
@@ -39,6 +31,9 @@ const styles = {
     fontWeight: "bold",
     textAlign: "left",
     color: "#000",
+    position: "relative",
```
Comment (NicholasLYang) [inline src/js/modules/core/components/SectionsButton.js]: Why?

File: src/js/modules/core/components/SignInButton.js [hunk 3]
```diff
@@ -10,10 +10,8 @@ const styles = {
     border: "solid 1.5px #dddddd",
     borderLeft: 0,
     minWidth: "66px",
-    minHeight: "45px",
     maxWidth: "80px",
-    maxHeight: "50px",
-    height: "10vh",
+    height: "45px",
```
Comment (NicholasLYang) [inline src/js/modules/core/components/SignInButton.js]: might as well do that for width

**Pull request merged**